src/components/Reinforcement/reinforcement.css

```diff
@@ -0,0 +1,34 @@
+.lil-reinforcement {
+  height: 408px;
+  background-color: #ffd240;
+  background-image: radial-gradient(at top right, #1778f9 30%, #ffd240 10%);
+  @apply lil-w-full lil-flex lil-items-center lil-rounded-xl;
+}
+
+.lil-reinforcement-content-wrapper {
+  @apply lil-px-12 lil-my-3 lil-justify-between;
+}
+
+.lil-reinforcement-title {
+  color: #4e3b00;
+  @apply lil-text-s-48 lil-text-left lil-font-bold;
+}
+
+@screen sm {
+  .lil-reinforcement {
+    background: #ffd240;
```

Comment:
not needed

Reply:
changed to background-image: none; as per suggestion